AWSPinpointTests/AWSPinpointTargetingClientTests.m

```diff
@@ -107,27 +110,15 @@ - (void)initializePinpointWithConfiguration:(AWSPinpointConfiguration *)configur
     self.pinpoint = [AWSPinpoint pinpointWithConfiguration:configuration];
 }
 
-- (void)initializePinpointWithConfiguration:(AWSPinpointConfiguration *)configuration {
-    [self initializePinpointWithConfiguration:configuration forceCreate:YES];
-}
-
-- (void)initializeMockApplicationWithOptOut:(BOOL)optOut {
+- (void)initializeMockApplicationWithRemoteNotifications:(BOOL)withRemoteNotifications {
     id mockApplication = OCMClassMock([UIApplication class]);
 
     OCMStub([mockApplication sharedApplication]).andReturn(mockApplication);
 
-    [self setSystemLevelOptOut:mockApplication withOptOut:optOut];
-}
-
-- (void)setSystemLevelOptOut:(UIApplication *)application withOptOut:(BOOL)optOut {
-    UIUserNotificationType notificationType = optOut ? UIUserNotificationTypeNone : UIUserNotificationTypeAlert;
+    UIUserNotificationType notificationType = withRemoteNotifications ? UIUserNotificationTypeNone : UIUserNotificationTypeAlert;
```

Comment:
if `withRemoteNotifications` == true, then I would expect notificationType to equal `UIUserNotificationTypeAlert`? but that doesn't seem to be the case.  Maybe we can chat offline about this?

Reply:
nice catch, i changed swapped the logic here from 
`OCMStub([application isRegisteredForRemoteNotifications]).andReturn(optOut == NO);`
to `OCMStub([mockApplication isRegisteredForRemoteNotifications]).andReturn(withRemoteNotifications);` but forgot to do it for `UIUserNotificationType notificationType = withRemoteNotifications ? UIUserNotificationTypeNone : UIUserNotificationTypeAlert;`